utils.js

```diff
@@ -12,20 +12,30 @@ import * as Config from 'resource:///org/gnome/shell/misc/config.js';
 
 import { Lib } from './imports.js';
 
-const Display = global.display;
+const Display = getGlobal().display;
 export let version = Config.PACKAGE_VERSION.split('.').map(Number);
 
 let warpRipple;
 
 let signals, touchCoords;
 let inTouch = false;
 
+/**
+ * Convenience function to get global.  Avoids linter flagging
+ * references to global as no-undef
+ * @returns global
+ */
+export function getGlobal() {
+    // eslint-disable-next-line no-undef
+    return global;
+}
```

Comment:
I don't think this is needed just to fix a lint warning. For eslint you should be able to fix the warning by adding `global` to the list of globals configured in `.eslintrc.yml`.

But also I haven't gotten the `.eslint.yml` to work for me. I have only ever worked with js configs for eslint.

Reply:
Well that's much easier!  I got sick of individually ignoring (with eslint ignore lines) every `global` reference... so thought to do it one place.

Will update the lint config.

Reply:
For reference, I'm just using https://open-vsx.org/extension/dbaeumer/vscode-eslint within `Code` for development.

I just installed it, did a `npm install -g eslint`, closed/opened `Code` and then added the PaperWM folder to the workspace.